src/torchcodec/decoders/_audio_decoder.py

```diff
@@ -25,10 +26,13 @@ class AudioDecoder:
     Returned samples are float samples normalized in [-1, 1]
 
     Args:
-        source (str, ``Pathlib.path``, ``torch.Tensor``, or bytes): The source of the audio:
+        source (str, ``Pathlib.path``,
+                ``io.RawIOBase``, ``io.BufferedReader``,
+                bytes, or ``torch.Tensor``): The source of the audio:
```

Comment:
Will this still render okay? I wanted to keep line length under control.

Reply:
I'm not sure, we'll have to check the built docs before merging. But I don't think we should always stick to actual types in the documentation, because they're not always the most informative. Anecdotally, I personally have no idea what ``io.RawIOBase`` or ``io.BufferedReader`` are, so reading these types in the doc doesn't tell me much about what I can and cannot do.

I think it would be much more informative if we were to document instead something like:

```py
source (str, ``Pathlib.path``, bytes, ``torch.Tensor``, or file-like object): 
```

and then describe what we mean by "file-like" later in the docstring.